Prevent voting in SCHEDULE_PUBLISHED conferences

smtchahal opened the pull request:
Prevent voting only in those proposals whose conferences have the status
SCHEDULE_PUBLISHED.

Relevant: #372

kracekumar commented on junction/proposals/votes_views.py:
```diff
@@ -113,8 +116,9 @@ def proposal_reviewer_vote(request, conference_slug, proposal_slug):
 
-    if not permissions.is_proposal_section_reviewer(request.user,
-                                                    conference, proposal):
+    if (not permissions.is_proposal_section_reviewer(request.user,
```
Two `not` can be reduced to one right ?

kracekumar commented:
Can you pull latest master ?

kracekumar commented:
Just noticed, can you add test case ?

smtchahal commented:
Sure. I'm on it.

smtchahal commented:
Actually, I'm kind of stuck at something and I can't figure it out :disappointed:.

The tests I need to add need access to a proposal that was created in a conference with the status "schedule published" status, but those don't exist in fixtures. So I edited the `create_proposals` function so it returns a dictionary, similar to the `conferences` fixture, returning a `past` and a `future` proposal. Then I wrote the tests and they ran successfully, as expected.

The problem is, the tests ran fine even when I removed the code I added in `vote_views` that prevented voting on "schedule published" conference proposals. See https://github.com/smtchahal/junction/commit/ee3b7c578d4a12343f14aaf14ad37a2ebaeec41f.
